docs/synthetics/ci-cd/github-actions.md

```diff
@@ -1,13 +1,30 @@
 title: GitHub Actions Integration
 description: Guide on how to integrate Sematext Synthetics with GitHub Actions.
 
+## Configure repository variables for the integration
+The GitHub Actions integration works by running a Job which runs a Synthetics monitor of your choice against the desired URL. In order to specify which monitor this will be, as well as the API key required to run it, you need to configure some repository variables first.
+
 ### Create API Key Secret
 
 The first step is to create a secret from the repository's **Settings** page for the Sematext API Key, which is needed to authenticate the request to run your desired Synthetics monitor. Go to your repository's **Settings**, and then in the sidebar under **Security** click **Secrets and variables** and then **Actions**. Create a new repository secret and name it `SEMATEXT_API_KEY`, then paste in the [API key for your account](./overview.md#finding-your-sematext-cloud-accounts-api-key).
 
 ![CI/CD GitHub Secret](./images/ci-cd-github-secret.png)
 
-### Add the Run Monitor Job to your GitHub Action Workflow
+
+### Create Repository Variables for monitor information
```

Comment:
Mixed casing style?

Reply:
I wanted to highlight "Repository Variables" since it's a special term, but I'll capitalize `Monitors` as well